inst/rmarkdown/templates/vaccination/skeleton/skeleton.Rmd

```diff
@@ -655,16 +687,25 @@ year age group.
 ```{r cluster_hh_size}
 # get counts of the number of households per cluster
 clustersize <- study_data_cleaned %>% 
-  count(cluster)
+  group_by(cluster) %>% 
+  # return the number of unique household IDs by each cluster
+  summarise(
+    n = length(
+      unique(q14_hh_no)
+      )
+    )
 
 # get the median number of households per cluster
 clustermed <- clustersize %>% 
-  pull(n) %>%
-  median()
+  summarise(
```

Comment:
These are variables that are printed inline. They should be single strings or numbers. Here, you are making them data frames.

Reply:
... dunno... worked when knitted...